worlds/generic/docs/setup_en.md

```diff
@@ -47,39 +50,142 @@ slot name and, if the relevant game requires it, files to associate it with that
-installation directory of the Archipelago installation and placing each YAML file in the `Players` folder. If the folder
-does not exist then it can be created manually.
 
-After filling the `Players` folder the `ArchipelagoGenerate.exe` program should be run in order to generate a
-multiworld. The output of this process is placed in the `output` folder.
+#### On your local installation
 
-#### Changing local host settings for generation
+It is possible to generate the multiworld locally, using a local Archipelago installation. This is done by entering the
+Archipelago installation folder (usually C:\ProgramData\Archipelago) and placing each YAML file in the `Players` folder.
+If the folder does not exist then it can be created manually. The files here should not be compressed.
+
+After filling the `Players` folder, the `ArchipelagoGenerate.exe` program should be run in order to generate a
+multiworld. The output of this process is placed in the `output` folder (usually named something like `AP_XXXXX.zip`).
+
+##### Changing local host settings for generation
 
 Sometimes there are various settings that you may want to change before rolling a seed such as enabling race mode,
 auto-release, plando support, or setting a password.
 
-All of these settings plus other options are able to be changed by modifying the `host.yaml` file in the Archipelago
+All of these settings, plus other options, may be changed by modifying the `host.yaml` file in the Archipelago
 installation folder. The settings chosen here are baked into the `.archipelago` file that gets output with the other
-files after generation so if rolling locally ensure this file is edited to your liking *before* rolling the seed.
+files after generation so if rolling locally ensure this file is edited to your liking **before** rolling the seed. This
+file is overwritten when running the Archipelago Installation software. If you have changed settings in this file, and
+would like to retain them, you may rename the file to `options.yaml`.
 
 ## Hosting an Archipelago Server
 
-The output of rolling a YAML will be a `.archipelago` file which can be subsequently uploaded to the Archipelago host
-game page. Archipelago host game page: [Archipelago Seed Upload Page](/uploads)
-
-The `.archipelago` file may be run locally in order to host the multiworld on the local machine. This is done by
-running `ArchipelagoServer.exe` and pointing the resulting file selection prompt to the `.archipelago` file that was
-generated.
\ No newline at end of file
+When a multiworld seed is generated, the multidata will be output as a `.archipelago`. If the game was generated locally,
+a compressed folder will be in `/output` and will contain the `.archipelago`, the spoiler, and any relevant files for the
+generated games.
+
+### Hosting on the website
+
+After a seed page has been created on the website, clicking on `Create Room` will create a new server instance, and a
```

Comment:
Maybe add a note here to explain that created rooms can be found in user-content w/ a link?